### ISSUE TITLE
Docker taking up huge disk space

### ISSUE THREAD

**Issue by jathifrusl** [mailcow/mailcow-dockerized#442]
Hi! I have noticed that docker seems slowly taking a lot of disk space from my vps. how do i fix this?
![untitled](https://user-images.githubusercontent.com/30011328/27990141-f5c57cf0-647f-11e7-8e43-f9754cfe6b0a.png)
Besides mailcow, I have nothing in docker.

furthermore, I have executed `docker system df -v` and I saw several redundant repository containers of different sizes and tags most of which take hundreds of megabytes in space and I guess those are the culprit. how to do I remove the unused or useless ones? thank you!

could I just remove all files in `/var/lib/docker/aufs/diff` and re-pull all images with `docker-compose pull`?

update:
executing this didn't solve the problem.
```
docker rmi -f $(docker images -f "dangling=true" -q)
docker volume rm $(docker volume ls -qf dangling=true)
```

**Comment by PhoenixPeca** [comment]
Hello @jathifrusl!

**This issue has been fixed in bdfaef0 just yesterday!**

To reclaim the lost disk space, please execute the `./update.sh` script and it will automatically find and delete the old image tags. But, if you're still unsatisfied, you may clean up your system with `docker system prune` but please do make sure all containers are up and running before doing such.

If you're still experiencing this issue, please follow up. 😉

**Comment by jathifrusl** [comment]
@PhoenixPeca thank you!
here's my disk stat. it really helps a lot.
![untitled](https://user-images.githubusercontent.com/30011328/27990410-d41fbd6e-6488-11e7-858d-170fafd97c62.png)
